syncano_cli/execute/utils.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+import json
+
+from syncano_cli.logger import get_logger
+
+LOG = get_logger('syncano-execute')
+
+
+def print_response(response):
```

Comment:
@opalczynski I don't understand You, what do You mean? I see no `response.result['response']` field.